provisioner/terraform/resources.go

```diff
@@ -3,6 +3,7 @@ package terraform
 import (
```

Comment:
Since you're modifying `resources.go`, should we modify `resources_test.go` as well? you could add use case to `testdata`

Reply:
done.